### PR TITLE
[Docs] rsz documentation

### PR DESCRIPTION
**Pull request by luarss**
Fixes missing functions and formatting. 

https://openroad--3701.org.readthedocs.build/en/3701/main/src/rsz/README.html

### CODE REVIEW

File: src/rsz/README.md [hunk 1]
```diff
@@ -7,66 +7,66 @@ utilization is exceeded.
...
 
-Use the `remove_buffers` command to remove buffers inserted by synthesis. This
-step is recommended before using `repair_design` so that there is more flexibility
-in buffering nets.
+| Switch Name | Description |
+| ----- | ----- |
+| `-layer` | select layer name |
+| `-via` | select via layer name. note that via resistance is per cut/via, not area based |
+| `-capacitance` | capacitance per unit length, same convention as `set_wire_rc` |
+| `-resistance` | resistance per unit length, same convention as `set_wire-rc` |
```
Comment (rovinski) [inline src/rsz/README.md:65]:
```suggestion
| `-resistance` | resistance per unit length, same convention as `set_wire_rc` |
```

File: src/rsz/README.md [hunk 2]
```diff
@@ -7,66 +7,66 @@ utilization is exceeded.
...
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-clock` | enable setting of RC for clock nets |
+| `-signal` | enable setting of RC for signal nets | 
+| `-layer` | if used, the LEF default RC values are used. |
+| `-resistance` | resistance per unit length, units are from the first Liberty file read, usually in the form of $\frac{resistanceUnit}{distanceUnit}$. Usually kΩ/µm. |
```
Comment (rovinski) [inline src/rsz/README.md:41]: Where is capacitance?

File: src/rsz/README.md [hunk 3]
```diff
@@ -7,66 +7,66 @@ utilization is exceeded.
...
 The `set_layer_rc` command can be used to set the resistance and capacitance
 for a layer or via. This is useful if these values are missing from the LEF file,
 or to override the values in the LEF.
 
+```tcl
+set_layer_rc 
+    [-layer layer]
+    [-via via_layer]
+    [-capacitance cap]
+    [-resistance res]
```
Comment (rovinski) [inline src/rsz/README.md:54]: Might as well order it resistance then capacitance to match order of `set_wire_rc`

File: src/rsz/README.md (hunk 1 above)
Comment (rovinski) [reply src/rsz/README.md:65]: Also same note about ordering

File: src/rsz/README.md [hunk 4]
```diff
@@ -7,66 +7,68 @@ utilization is exceeded.
...
+    [-capacitance cap]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-clock` | Enable setting of RC for clock nets. |
+| `-signal` | Enable setting of RC for signal nets. | 
+| `-layer` | If `-layer` is used, the LEF technology resistance and area/edge capacitance values for the layer are used for a default width wire on the layer. |
```
Comment (vvbandeira) [inline src/rsz/README.md:40]:
```suggestion
| `-layer` | Use the LEF technology resistance and area/edge capacitance values for the layer. This is used for a default width wire on the layer. |
```

File: src/rsz/README.md [hunk 5]
```diff
@@ -7,66 +7,68 @@ utilization is exceeded.
...
-Use the `remove_buffers` command to remove buffers inserted by synthesis. This
-step is recommended before using `repair_design` so that there is more flexibility
-in buffering nets.
+| Switch Name | Description |
+| ----- | ----- |
+| `-layer` | Set **routing** layer name to modify. |
+| `-via` | Select via layer name. note that via resistance is per cut/via, not area based. |
+| `-resistance` | Resistance per unit length, same convention as `set_wire_rc`. |
+| `-capacitance` | Capacitance per unit length, same convention as `set_wire_rc`. |
+| `-corner` | Process corner to use. |
```
Comment (vvbandeira) [inline src/rsz/README.md:68]:
```suggestion
| `-layer` | Set layer name to modify. Note that the layer must be a routing layer. |
| `-via` | Select via layer name. Note that via resistance is per cut/via, not area-based. |
| `-resistance` | Resistance per unit length, same convention as `set_wire_rc`. |
| `-capacitance` | Capacitance per unit length, same convention as `set_wire_rc`. |
| `-corner` | Process corner to use. |
```

File: src/rsz/README.md [hunk 6]
```diff
@@ -79,158 +81,245 @@ After the `global_route` command has been called, the global routing topology
...
+#### Options
 
-```
-set_max_fanout <fanout> [current_design]
-```
+| Switch Name | Description |
+| ----- | ----- |
+| `-max_wire_length` | Maximum length of wires (in microns), defaults to a value that minimizes the wire delay for the wire RC values specified by `set_wire_rc`. |
+| `-slew_margin` | Add a slew margin (0-100, integer). |
+| `-cap_margin` | Add a capactitance margin (0-100, integer). |
```
Comment (vvbandeira) [inline src/rsz/README.md:174]: Default and range formatting.

File: src/rsz/README.md [hunk 7]
```diff
@@ -79,158 +81,245 @@ After the `global_route` command has been called, the global routing topology
...
 
-The `report_floating_nets` command reports nets with only one pin connection.
-Use the `-verbose` flag to see the net names.
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-verbose` | Print the net names. |
+
+### Useful developer functions
```
Comment (vvbandeira) [inline src/rsz/README.md:282]:
```suggestion
### Useful Developer Commands
```

File: src/rsz/README.md [hunk 8]
```diff
@@ -79,158 +81,245 @@ After the `global_route` command has been called, the global routing topology
...
+
+If you are a developer, you might find these useful. More details can be found in the [source file](./src/Resizer.cc) or the [swig file](./src/Resizer.i).
+
+```tcl
+# repair setup pin 
+repair_setup_pin end_pin
```
Comment (vvbandeira) [inline src/rsz/README.md:288]: Table

File: src/rsz/README.md [hunk 9]
```diff
@@ -264,11 +353,19 @@ report_wns
 
 ## Regression tests
 
+There are a set of regression tests in `/test`. For more information, refer to this [section](../../README.md#regression-tests).
```
Comment (vvbandeira) [inline src/rsz/README.md:356]:
```suggestion
There are a set of regression tests in `./test`. For more information, refer to this [section](../../README.md#regression-tests). 
```

**Review by vvbandeira — CHANGES_REQUESTED** [review]
PR has merge conflict

**Review by vvbandeira — APPROVED** [review]

**Pull request merged**